pkg/framework/spi/crypto/api.go

```diff
@@ -0,0 +1,17 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package crypto
```

Comment:
I wonder if we really need an extra folder under spi?

Reply:
moved file to spi.go under framework ... the SPIs interfaces are internal to the framework and should not be visible outside of it.